.gitignore

```diff
@@ -33,3 +33,7 @@ csr.pem
 
 .DS_STORE
 
+# Ensuring no images and video are commited in ./video and ./image directories
+
+
+markdown/*
```

Comment:
Why have you added the markdown directory here?
It should contain the `videos` and `images` directories that get the uploaded files.